src/black/parsing.py

```diff
@@ -102,8 +102,7 @@ def lib2to3_parse(src_txt: str, target_versions: Iterable[TargetVersion] = ()) -
 
 def lib2to3_unparse(node: Node) -> str:
     """Given a lib2to3 node, return its string representation."""
-    code = str(node)
-    return code
+    return str(node)
```

Comment:
just a suggestion - perhaps make separate PRs, one for anything totally non-controversial (like this), and another for things like factoring out `_extracted_from_maybe_make_parens_invisible_in_atom_28`?

Reply:
Sorry for the in `atom_28`, this was a passed issue!
I didn't know that you prefer separated PRs.  Go it

Reply:
I'm not a maintainer so it's not my call, but in general small and focused PRs are easier to review